public/css/index.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background: #1e1e25;
+}
+
+.form {
+  position: absolute;
+  left: 0;
+  right: 0;
```

Comment:
`absolute` - to jest ostatnio deska ratunku a nie główne narzędzie do tworzenia layotów :)
nie korzystaj z niego jeśli możesz coś zrobić za pomocą flexa.
czy to jest wogóle tutaj potrzebne? :)